### ISSUE TITLE
Allow default parameters to web flow in okta.yaml

### ISSUE THREAD

**Issue by Liz4v** [okta/okta-aws-cli#210]
As far as I can tell from the documentation, the only way to let my team see the list of idps is to force them to fill up `org-domain` and `oidc-client`. Sure that's scriptable, but I would prefer to give my team a config file instead, so that the command:

```
okta-aws-cli web
```

Would be able to just bring up a list of id providers after authentication.

**Comment by EliseAv** [comment]
Presumably can be solved by pull request #200

**Comment by monde** [comment]
@EliseAv is this something different than configure by profile name?

https://github.com/okta/okta-aws-cli?tab=readme-ov-file#configuration-by-profile-name

**Comment by EliseAv** [comment]
Yes. The central problem is that these two behave differently and they should behave exactly the same:

```
okta-aws-cli web
okta-aws-cli web -p default
```

**Comment by monde** [comment]
Closed with #200
